appender.go

```diff
@@ -162,13 +166,13 @@ func TestAppender(t *testing.T, store types.Storager) {
 			})
 
 			var buf bytes.Buffer
-			_, err = store.Read(path, &buf, pairs.WithSize(size))
+			_, err = store.Read(path, &buf, pairs.WithSize(size*2))
 
 			Convey("Read error should be nil", func() {
 				So(err, ShouldBeNil)
 			})
 			Convey("The content should be match", func() {
-				So(sha256.Sum256(buf.Bytes()), ShouldResemble, sha256.Sum256(content))
+				So(sha256.Sum256(buf.Bytes()), ShouldResemble, sha256.Sum256(append(content, content...)))
```

Comment:
How about using `bytes.Repeat(content, 2)` here to make it more readable?

Reply:
thanks